bin/hoc-2015-oct-26-mailing/send-uk-poster-test-mails

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+
+./send-to-mailing-list 10-26-hoc-uk-posters hoc-2015-oct-26-mailing/test-params.csv
```

Comment:
in any case I don't see it in this directory in this checkin so it won't work ;)